zap/src/main/java/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -153,6 +155,8 @@ public int getValue() {
             new EnumMap<>(FontUtils.FontType.class);
     private Map<FontUtils.FontType, String> fontTypeLabels =
             new EnumMap<>(FontUtils.FontType.class);
+    private ZapNumberSpinner iconSize = null;
```

Comment:
Remove the null initialization?

Reply:
I didn't mean to dismiss this comment with my approval (I assumed this would be addressed either way), it's just I don't have additional comments.

Reply:
I didn’t wanna block on such a minor detail. I guess I could tackle it tonight too 😉

Reply:
Thanks for tackling it @kingthorin.